source/frontend/src/features/mapSideBar/compensation/detail/CompensationRequisitionDetailContainer.tsx

```diff
@@ -75,12 +78,22 @@ export const CompensationRequisitionDetailContainer: React.FunctionComponent<
             const organization = await getOrganization(compensation.interestHolder.organizationId);
             setPayeeOrganization(organization);
           }
+        } else if (compensation.compReqLeaseStakeholder?.length > 0) {
```

Comment:
perhaps ...Stakeholder(s) if that is an array.

Reply:
It's an array, but it will never be more than one. It makes more sense to me as Singular.